### PR TITLE
update with info about shentu v2.8.0

### PR DESCRIPTION
**Pull request by haozhan9**
for Shentu v2.8.0

### CODE REVIEW

File: shentu/chain.json [hunk 1]
```diff
@@ -35,47 +35,38 @@
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/ShentuChain/mainnet/main/shentu-2.2/genesis.json"
     },
-    "recommended_version": "v2.7.1",
+    "recommended_version": "v2.8.0",
```
Comment (JeremyParish69) [inline shentu/chain.json:38]: Please add v2.8.0 to the bottom of the `versions` array below

File: shentu/chain.json [hunk 2]
```diff
@@ -35,47 +35,38 @@
...
+      "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_arm64_macos",
+      "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_win_x86_64.exe"
     },
     "versions": [
       {
         "name": "v2.6.0",
         "height": 10485430,
         "proposal": 14,
         "tag": "v2.6.0",
-        "recommended_version": "v2.6.0",
```
Comment (JeremyParish69) [inline shentu/chain.json:58]: Please re-add the recommended and compatible versions back in (for all versions it was removed)

File: shentu/chain.json (hunk 1 above)
Comment (haozhan9) [reply shentu/chain.json:38]: added.

File: shentu/chain.json (hunk 2 above)
Comment (haozhan9) [reply shentu/chain.json:58]: added

**Review by tac0turtle — APPROVED** [review]

**Pull request merged**